src/eia930.py

```diff
@@ -173,7 +174,7 @@ def reformat_chalendar(raw):
     print("Filtering")
     cleaned = (
         raw.loc[:, target_cols]
-        .melt(ignore_index=False, value_name="generation")
+        .melt(ignore_index=False, value_name="generation", var_name="variable")
```

Comment:
@gailin-p why did you end up adding this argument if it didn't change anything downstream? I'm wondering if there is something here that could be messing with the downstream calculations?

Reply:
this just guarantees that the column name is what we're splitting in the next line; it doesn't change the function exports. Without this, 'melt' is sensitive to `raw.columns.name`, which can be different depending on where/how the raw data frame was imported